src/util/enums.ts

```diff
@@ -0,0 +1,13 @@
+export enum BuilderActionEnum {
+  Add = 'ADD',
+  Remove = 'REMOVE',
+  Update = 'UPDATE',
+  Reorder = 'REORDER',
+}
+
+export enum ConfigArrayEnum {
+  Fields = 'fields',
+  Operations = 'operations',
+  Displays = 'displays',
+  Constants = 'constants',
+}
```

Comment:
I had a question about this - because the compiler was very angry when we exported Enum

Reply:
Eh, show me when you get the chance

Reply:
Thanks for chatting about this earlier @LoneRifle . This is a workaround because it seems like the jest framework is unable to recognize enums exported from the `*.d.ts` files - we decided to go ahead with this workaround for now.